### PR TITLE
Display tooltip messages with style

### PR DESCRIPTION
**Pull request by josehernandezfintecheandomx**
## Description

Display tooltip messages with style with a bigger font size and using the base color 

## Screenshots, if any

Screenshot 2023-10-01 at 5 57 58 p m

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [ ] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/products/loan-products/loan-product-stepper/loan-product-currency-step/loan-product-currency-step.component.ts [hunk 1]
```diff
@@ -32,7 +32,9 @@ export class LoanProductCurrencyStepComponent implements OnInit {
   createLoanProductCurrencyForm() {
     this.loanProductCurrencyForm = this.formBuilder.group({
       'currencyCode': ['', Validators.required],
-      'digitsAfterDecimal': [2, Validators.required]
+      'digitsAfterDecimal': [2, Validators.required],
+      'inMultiplesOf': '',
```
Comment (adamsaghy) [inline src/app/products/loan-products/loan-product-stepper/loan-product-currency-step/loan-product-currency-step.component.ts:36]: Do we need this to be put back?

**Review by adamsaghy — APPROVED** [review]

File: src/app/products/loan-products/loan-product-stepper/loan-product-payment-strategy-step/payment-allocation-model.ts [hunk 2]
```diff
@@ -41,6 +41,12 @@ export class AdvancedPaymentAllocation {
   futureInstallmentAllocationRules: FutureInstallmentAllocationRule[];
 }
 
+export class AdvancePaymentAllocationData {
+  transactionTypes: PaymentAllocationTransactionType[];
```
Comment (adamsaghy) [inline src/app/products/loan-products/loan-product-stepper/loan-product-payment-strategy-step/payment-allocation-model.ts:45]: It would be better to move these changes into a separate PR. They are not really related to tooltips

**Review by galovics — APPROVED** [review]
